### DISCUSSION TITLE
Trying to remove I T administrator  and apps!!  HELP PLS

### DISCUSSION THREAD

**Discussion by Jcutty655** [community/community#80856] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Bug

### Body

Trying to remove I T administrator of my phone.  She has full control of my device and put 600 apps on phone.. also smart switched data without  touching device!! HELP PLS  i dont know anything about this coding pr development

**Comment by queenofcorgis** [comment]
Hi @Jcutty655 , Thanks for reaching out! 

We're sorry to hear you're experiencing this issue.

GitHub is a software hosting service, we host millions of projects, so probably every phone and computer on the planet has software on it that is also hosted here.

We're not able to provide support or answers for those individual projects. GitHub Support can only provide support for GitHub itself.

I'm sorry we can't help here but we'd recommend contacting a local IT specialist for assistance if you think your devices are affected by viruses or malware. They should be able to help you with these applications!
